controller-server/src/test/java/com/yahoo/vespa/hosted/controller/deployment/InternalDeploymentTester.java

```diff
@@ -137,26 +105,41 @@ public InternalDeploymentTester() {
         domain.services.put(ATHENZ_SERVICE, new AthenzDbMock.Service(true));
     }
 
+    public ReadyJobsTrigger readyJobsTrigger() {
+        return readyJobsTrigger;
+    }
+
+    public NameServiceDispatcher nameServiceDispatcher() {
+        return nameServiceDispatcher;
+    }
+
+    /** Returns the default deployment context owned by this */
+    public DeploymentContext deploymentContext() {
+        return defaultContext;
+    }
+
+    /** Create a new deployment context for given application */
+    public DeploymentContext newDeploymentContext(String tenantName, String applicationName, String instanceName) {
```

Comment:
It's possible we should leave the instance out here, and support it in the context somehow. Let's see how it turns out.